artiq/frontend/artiq_sinara_tester.py

```diff
@@ -352,6 +355,69 @@ def mirny_rf_switch_wave(self, channels):
             for channel in channels:
                 channel.pulse(100*ms)
                 delay(100*ms)
+    @kernel
+    def init_almazny(self, almazny):
+        self.core.break_realtime()
+        almazny.init()
+        almazny.output_toggle(True)
+
+    @kernel
+    def almazny_set_attenuators(self, almazny, ch, atts):
+        self.core.break_realtime()
+        almazny.set_att_mu(ch, atts)
+
+    @kernel
+    def almazny_set_attenuators_db(self, almazny, ch, atts):
```

Comment:
Convention is no suffix for the function with usual units and ``_mu`` for the one with hardware/machine units.
So it should be ``almazny_set_attenuators_mu`` and ``almazny_set_attenuators``.